cfgov/v1/jinja2/v1/events/_event-detail.html

```diff
@@ -80,7 +80,7 @@ <h1 class="u-visually-hidden">Event viewing details</h1>
     {% if page.tags.names() | length %}
     <footer>
         {%- import 'v1/includes/tags.html' as tags %}
-        {{ tags.render(page.related_metadata_tags()) }}
+        {{ tags.render(page.related_metadata_tags().links) }}
```

Comment:
Instead of repeating this three times, could we just adjust [this code](https://github.com/cfpb/consumerfinance.gov/blob/4ff90e5c67b7ac1f2e810a129e30b775df823fb4/cfgov/v1/models/base.py#L238) so that the Python code returns a list of dicts without the unnecessary `links` level?

Reply:
Discussed in chat, defer to a future PR.